packages/maeve-input/src/maeve-input.jsx

```diff
@@ -119,6 +119,7 @@ class MaeveInput extends React.Component {
           {...inputProps}
         />
         {dropdown}
+        {this.props.children}
```

Comment:
Second dropdown will show up only if it is defined within the `<MaeveInput />` tag.
As for the default dropdown, it wont show up untill you declare `autocomplete`.

Anyone can choose either of the options they like.